src/api/index.js

```diff
@@ -25,7 +25,9 @@ const getDSData = url => {
   return axios
     .get(url)
     .then(res => JSON.parse(res.data))
-    .catch(err => err);
+    .catch(err => {
+      console.log(err);
```

Comment:
Good catch changing it to output to the console, I do wonder perhaps it would be clearer as console.error. Nice work!